99-x52pro.rules

```diff
@@ -1,9 +1,9 @@
 #X52 Flight System
-ACTION=="add", SUBSYSTEMS=="usb", ATTRS{idVendor}=="06a3", ATTRS{idProduct}=="0255", MODE="660", GROUP="plugdev"
-ACTION=="add", SUBSYSTEMS=="usb", ATTRS{idVendor}=="06a3", ATTRS{idProduct}=="075c", MODE="660", GROUP="plugdev"
+ACTION=="add", SUBSYSTEMS=="usb", ATTRS{idVendor}=="06a3", ATTRS{idProduct}=="0255", TAG+="uaccess"
+ACTION=="add", SUBSYSTEMS=="usb", ATTRS{idVendor}=="06a3", ATTRS{idProduct}=="075c", TAG+="uaccess"
 
 #X52 Pro Flight System
-ACTION=="add", SUBSYSTEMS=="usb", ATTRS{idVendor}=="06a3", ATTRS{idProduct}=="0762", MODE="660", GROUP="plugdev"
+ACTION=="add", SUBSYSTEMS=="usb", ATTRS{idVendor}=="06a3", ATTRS{idProduct}=="0762", TAG+="uaccess"
 
 #Pro Flight Yoke System
-ACTION=="add", SUBSYSTEM=="input", ATTR{idVendor}=="06a3", ATTR{idProduct}=="0bac", MODE="660", GROUP="plugdev"
+ACTION=="add", SUBSYSTEM=="input", ATTR{idVendor}=="06a3", ATTR{idProduct}=="0bac", TAG+="uaccess"
```

Comment:
Please also fix this entry as I have already done in 7d7d27c17ee25f85b82ed37e190e1f9f8937d636